src/sidebar/components/test/selection-tabs-test.js

```diff
@@ -87,28 +96,105 @@ describe('selectionTabs', function () {
+        assert.notInclude(msg.textContent, 'Create one by selecting some text and clicking the');
       });
 
-      var newNoteElem = elem[0].querySelectorAll('new-note-btn');
-      assert.equal(newNoteElem.length, 0);
+      it('should display the shorter version of the no annotations message when there are no annotations', function () {
+        var elem = util.createDirective(document, 'selectionTabs', {
+          selectedTab: 'annotation',
+          totalAnnotations: 0,
+          totalNotes: '10',
+        });
+        var msg = elem[0].querySelectorAll('.annotation-unavailable-message__label')[0];
+
+        assert.include(msg.textContent, 'There are no annotations in this group.');
+        assert.notInclude(msg.textContent, 'Create one by clicking the');
+        assert.notInclude(msg.textContent, 'Create one by selecting some text and clicking the');
+      });
+    });
+
+    context('when the sidebar tutorial is not displayed', function () {
+      fakeSession.state.preferences.show_sidebar_tutorial = false;
+      fakeSettings.enableExperimentalNewNoteButton = false;
+
+      it('should display the longer version of the no notes message when there are no notes', function () {
+        var elem = util.createDirective(document, 'selectionTabs', {
+          selectedTab: 'note',
+          totalAnnotations: '10',
+          totalNotes: 0,
+        });
+        var unavailableMsg = elem[0].querySelectorAll('.annotation-unavailable-message__label')[0];
+        var unavailableTutorial = elem[0].querySelectorAll('.annotation-unavailable-message__tutorial')[0];
+        var noteIcon = unavailableTutorial.querySelectorAll('i')[0];
+
+        assert.include(unavailableMsg.textContent, 'There are no page notes in this group.');
+        assert.include(unavailableTutorial.textContent, 'Create one by clicking the');
+        assert.isTrue(noteIcon.classList.contains('h-icon-note'));
+      });
+
+      it('should display the longer version of the no annotations message when there are no annotations', function () {
+        var elem = util.createDirective(document, 'selectionTabs', {
+          selectedTab: 'annotation',
+          totalAnnotations: 0,
+          totalNotes: '10',
+        });
+        var unavailableMsg = elem[0].querySelectorAll('.annotation-unavailable-message__label')[0];
+        var unavailableTutorial = elem[0].querySelectorAll('.annotation-unavailable-message__tutorial')[0];
+        var noteIcon = unavailableTutorial.querySelectorAll('i')[0];
+
+        assert.include(unavailableMsg.textContent, 'There are no annotations in this group.');
+        assert.include(unavailableTutorial.textContent, 'Create one by selecting some text and clicking the');
+        assert.isTrue(noteIcon.classList.contains('h-icon-annotate'));
+      });
+
+      it('should not display the new note button when the notes tab is active and the new note button is disabled', function () {
```

Comment:
These tests are in a `when the sidebar tutorial is not displayed` context block, but they apply whether or not the sidebar tutorial is shown, don't they? If so, then they should be outside the context block. Also, don't we already have tests for this?